src/jupyter_contrib_nbextensions/nbextensions/comment-uncomment/readme.md

```diff
@@ -2,3 +2,8 @@ Comment/Uncomment
 =================
 
 Adds a new configurable hotkey binding to toggle comments on/off.
+
+Options
+-------
+
+indent: add comment at current indent level instead at beginning of line
```

Comment:
I think this should be 

> instead of at the beginning of the line

?

Might also be worth givin the full name of `comment_uncomment_indent`.